core/foundation/inc/ROOT/RWrap_libcpp_string_view.h

```diff
@@ -425,7 +425,7 @@ inline namespace __1 {
          __pos += __n;
       else
          __pos = __sz;
-      const _CharT* __r = _VSTD::__find_end(
+      const _CharT* __r = _VSTD::find_end(
```

Comment:
@pcanal and @Axel-Naumann, MSVC + modules does not expose `_VSTD::__find_end`. I find the replacement suitable. Let us know if you have any comments.

Reply:
Fair enough.